blockchain/filedao/filedao_v2_util.go

```diff
@@ -206,13 +206,9 @@ func (fd *fileDAOv2) getBlockStore(height uint64) (*block.Store, error) {
 
 	// add to read cache
 	fd.blkCache.Add(storeKey, pbStores)
-	return extractBlockStore(pbStores, stagingKey(height, fd.header))
+	return extractBlockStore(fd.deser, pbStores, stagingKey(height, fd.header))
 }
 
-func extractBlockStore(pbStores *iotextypes.BlockStores, height uint64) (*block.Store, error) {
-	info := &block.Store{}
-	if err := info.FromProto(pbStores.BlockStores[height]); err != nil {
-		return nil, err
-	}
-	return info, nil
+func extractBlockStore(deser *block.Deserializer, pbStores *iotextypes.BlockStores, height uint64) (*block.Store, error) {
+	return deser.FromBlockStoreProto(pbStores.BlockStores[height])
```

Comment:
can remove this func, b/c it is just 1 line code

Reply:
done